src/java/main/org/apache/zookeeper/ZooDefs.java

```diff
@@ -74,12 +74,16 @@
 
         public final int createTTL = 21;
 
+        public final int addPersistentWatch = 22;
+
         public final int auth = 100;
 
         public final int setWatches = 101;
 
         public final int sasl = 102;
 
+        public final int setWatches2 = 103;
```

Comment:
I see - sure that's OK with me. Frankly, anything to get this PR moving. It's been almost impossible to get a ZK committer interested in this and I see this as a vital addition to ZK.